src/sentry/static/sentry/app/views/alerts/list/index.tsx

```diff
@@ -60,11 +61,63 @@ class IncidentsList extends AsyncComponent<Props, State & AsyncComponent['state'
       ],
     ];
   }
+  onLoadAllEndpointsSuccess() {
+    const {incidentList} = this.state;
+    if (incidentList.length === 0) {
+      // Check if they have rules or not, to know which empty state message to display
+      const {params} = this.props;
+
+      this.setState({
+        loading: true,
+      });
+
+      this.api.request(`/organizations/${params && params.orgId}/alert-rules/`, {
+        method: 'GET',
+        success: data => {
+          this.setState({
+            hasAlertRule: data.length > 0 ? true : false,
+            loading: false,
+          });
+        },
+      });
+    }
+  }
```

Comment:
Can we use `requestPromise` + `async/await` here?

I would also add some error handling